packages/shared/src/permissions/permissions.constants.ts

```diff
@@ -0,0 +1,523 @@
+import {
+  DefaultMemberRole,
+  Permission,
+  Role,
+} from "back-end/types/organization";
+
+export const POLICIES = [
+  "ReadData",
+  "Comments",
+  "FeaturesFullAccess",
+  "ArchetypesFullAccess",
+  "FeaturesBypassApprovals",
+  "ExperimentsFullAccess",
+  "VisualEditorFullAccess",
+  "SuperDeleteReports",
+  "DataSourcesFullAccess",
+  "DataSourceConfiguration",
+  "RunQueries",
+  "MetricsFullAccess",
+  "FactTablesFullAccess",
+  "FactMetricsFullAccess",
+  "FactFiltersFullAccess",
+  "DimensionsFullAccess",
+  "SegmentsFullAccess",
+  "IdeasFullAccess",
+  "PresentationsFullAccess",
+  "SDKPayloadPublish",
+  "SDKConnectionsFullAccess",
+  "AttributesFullAccess",
+  "EnvironmentsFullAccess",
+  "NamespacesFullAccess",
+  "SavedGroupsFullAccess",
+  "GeneralSettingsFullAccess",
+  "NorthStarMetricFullAccess",
+  "TeamManagementFullAccess",
+  "ProjectsFullAccess",
+  "TagsFullAccess",
+  "APIKeysFullAccess",
+  "IntegrationsFullAccess",
+  "EventWebhooksFullAccess",
+  "BillingFullAccess",
+  "AuditLogsFullAccess",
+  "CustomRolesFullAccess",
+] as const;
+
+export type Policy = typeof POLICIES[number];
+
+export const POLICY_PERMISSION_MAP: Record<Policy, Permission[]> = {
+  ReadData: ["readData"],
+  Comments: ["readData", "addComments"],
+  FeaturesFullAccess: [
+    "readData",
+    "manageFeatureDrafts",
+    "manageFeatures",
+    "manageArchetype",
+    "canReview",
+  ],
+  ArchetypesFullAccess: ["readData", "manageArchetype"],
+  FeaturesBypassApprovals: [
+    "readData",
+    "manageFeatureDrafts",
+    "manageFeatures",
+    "canReview",
+    "bypassApprovalChecks",
+  ],
+  ExperimentsFullAccess: ["readData", "createAnalyses"],
+  VisualEditorFullAccess: ["readData", "manageVisualChanges"],
+  SuperDeleteReports: ["readData", "superDeleteReport"],
+  DataSourcesFullAccess: [
+    "readData",
+    "createDatasources",
+    "editDatasourceSettings",
+  ],
+  DataSourceConfiguration: ["readData", "editDatasourceSettings"],
+  RunQueries: ["readData", "runQueries"],
+  MetricsFullAccess: ["readData", "createMetrics"],
+  FactTablesFullAccess: [
+    "readData",
+    "manageFactTables",
+    "manageFactMetrics",
+    "manageFactFilters",
+  ],
+  FactMetricsFullAccess: ["readData", "manageFactMetrics", "manageFactFilters"],
```

Comment:
@jdorn Added this and the `FactFiltersFullAccess` below. Wanted to call it out this is new.

Reply:
I would keep fact metrics and filters together in a single policy and not break them out.  They are very intertwined in the UI, so having them separate would be kind of weird.  We can always do it later if people ask for it.